### DISCUSSION TITLE
Toggle kana type

### DISCUSSION THREAD

**Discussion by h-youhei** [google/mozc#914] [Ideas]
半角カタカナは、まれにしか使いません。
なので、ひらがな・全角カタカナ・半角カタカナ間の切り替えの他に、ひらがな・全角カタカナ間の切り替えを用意してほしいです。

I want 'toggle kana type', that is toggle between hiragana and full-width katakana, in addition to 'Set input mode to next kana type', that is switch among hiragana, full-width katakana and half-width katakana.
That's because half-width katakana is rarely used.

**Comment by h-youhei** [comment]
また、入力モードの切り替えに加えて、変換についても同様の機能があるといいです。
具体的には、ひらがなモード時には全角カタカナに変換され、半角及び全角カタカナモード時にはひらがなに変換される機能です。

I also want 'toggle kana type' but to convert.
In particular, it convert to full-width katakana from hiragana, to hiragana from either katakana.

**Comment by haruomaki** [comment]
> I also want 'toggle kana type' but to convert.

変換中に使える「ひらがな・カタカナを切替」(`SwitchKanaType`) というコマンドが既にあるものの、これも実際は「ひらがな → 全角カタカナ → 半角カタカナ」とトグルしちゃいますね。
